starter-code/index.html

```diff
@@ -3,40 +3,82 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans|Roboto" rel="stylesheet">
 </head>
+
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+<div class="main-container">
+  <div class="flex1-container">
+    <div class="navbar">
+      <div id="brandLogo"></div>
+      <div class="navButtons">
+        <a href=""><i class="fa fa-premium"></i>Premium</a>
+        <a href=""><i class="fa fa-discover"></i>Discover</a>
+        <a href=""><i class="fa fa-help"></i>Help</a>
+        <a href=""><i class="fa fa-download"></i>Download</a>
+      </div>
+     </div>
+
+    <div class="backPic">
+      <h1 class="backPic-h1">Music for everyone.</h1>
+      <h3 class="backPic-h3">Spotify is now free on mobile, tablet and computer.</br>
+        Listen to the right music, wherever you are.</h3>
+    </div>
+    <div class="h2-center"><p>What’s on Spotify?</p></div>
+  </div>
+
+<!-- CONTENT -->
+  <div class="flex2-container">
```

Comment:
Another way of doing this is to use a list, it will allow you to add the image in your html and reduce your css : 

```html

      <ul>
        <li>
          <img src="images/music-icon.png">
          <h3>Millions of Songs</h3>
          <p> There are millions of
            <br>songs on Spotify</p>
        </li>
        <li>
          <img src="images/high-quality-icon.png">
          <h3>HD Music</h3>
          <p>Listen to music as if you
            <br> were listening live</p>
        </li>
        <li>
          <img src="images/devices-icon.png">
          <h3>Stream Everywhere</h3>
          <p>Stream music on your
            <br>smartphone, tablet or
            <br> computer
          </p>
        </li>
      </ul>
    </div>
```